src/routes/+layout.svelte

```diff
@@ -4,23 +4,27 @@
     import Footer from "$lib/components/Footer.svelte";
     import { browser } from "$app/environment";
 
-    // Google Analytics Script
-    function gtag() {
-        window.dataLayer.push(arguments);
-    }
+    // Google Analytics
     if (browser) {
+        // @ts-ignore
+        window.gtag = function () {
+            window.dataLayer.push(arguments);
+        };
         window.dataLayer = window.dataLayer || [];
-        gtag('js', new Date());
-        gtag('config', 'G-LLGRDWVVEV');
+        gtag("js", new Date());
+        gtag("config", "G-LLGRDWVVEV");
     }
 </script>
 
 <svelte:head>
-    <script async src="https://www.googletagmanager.com/gtag/js?id=G-LLGRDWVVEV"></script>
+    <script
+        async
+        src="https://www.googletagmanager.com/gtag/js?id=G-LLGRDWVVEV"
+    ></script>
```

Comment:
Did prettier do this? If so we need to add a `.prettierrc` file because the default line length is 70 which is way too short

Reply:
yeah it definitely did, i did not do this lol. will check